includes/functions.php

```diff
@@ -72,117 +66,3 @@ function render_theme_stories() {
 	//phpcs:disable WordPress.Security.EscapeOutput.OutputNotEscaped
 	echo get_plugin_instance()->customizer->render_stories();
 }
-
-/**
- * Returns field state for the provided view type.
- *
- * @since 1.5.0
- * @access private
- * @todo Move out of this file
- *
- * @param string $view View Type.
- *
- * @return GridView|ListView|CircleView|CarouselView
- */
-function get_field( $view = 'grid' ) {
```

Comment:
This is probably better suited as a method in a FieldState factory or something.

Reply:
Should I create a new FieldState factory? This would only be used for single use that `get_field` would be used in.

Reply:
Maked to a factory now.